whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/NrtmVersionInfoRepository.java

```diff
@@ -130,7 +146,7 @@ public NrtmVersionInfo findLastSnapshotVersion(final NrtmSource source) {
 
     public Optional<NrtmVersionInfo> findById(final long versionId) {
         final String sql = """
-            SELECT v.id, src.name, v.version, v.session_id, v.type, v.last_serial_id, v.created
+            SELECT v.id, src.id, src.name, v.version, v.session_id, v.type, v.last_serial_id, v.created
             FROM version_info v
```

Comment:
This will throw an emptyResultException optional.nullable has no effect?

Reply:
In that case catch the empty result exception and throw IllegalStateException or InternalException?My point is optional.empty will never be a case here as the method will throw exception, so no point in using optional actually

Reply:
done

Reply:
Here I feel more comfortable with try catch...you never know what could happen in the future